src/runtime/mod.rs

```diff
@@ -277,7 +277,8 @@ mod tests {
     fn test_canonicalize_rootfs() {
         let rootfs_name = "rootfs";
         let bundle = tempfile::tempdir().expect("failed to create tmp test bundle dir");
-        let rootfs_absolute_path = bundle.path().join(rootfs_name);
+        let bundle = fs::canonicalize(bundle.path()).expect("failed to canonicalize bundle");
```

Comment:
I wonder why macOS needs `fs::canonicalize()`.

Reply:
```
$ echo $TMPDIR; realpath $TMPDIR
/var/folders/_h/j_17023n23s3_50cq_gwhrrc0000gq/T/
/private/var/folders/_h/j_17023n23s3_50cq_gwhrrc0000gq/T
```

Reply:
May I ask you to add the comment?

Reply:
Done.